modules/ROOT/pages/migration-from-7x.adoc

```diff
@@ -3,9 +3,665 @@
+
+.Before: {productname} {productmajorversion}.
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea#custom-toolbar-split-button',
+  toolbar: 'myButton',
+  menubar: false,
+  setup: (editor) => {
+    editor.ui.registry.addSplitButton('myButton', {
+      text: 'My Button',
+      icon: 'info',
+      tooltip: 'This is an example split-button',
+      onAction: () => editor.insertContent('<p>You clicked the main button</p>'),
+      onItemAction: (api, value) => editor.insertContent(value),
+      fetch: (callback) => {
+        const items = [
+          {
+            type: 'choiceitem',
+            icon: 'notice',
+            text: 'Menu item 1',
+            value: '&nbsp;<em>You clicked menu item 1!</em>'
+          },
+          {
+            type: 'choiceitem',
+            icon: 'warning',
+            text: 'Menu item 2',
+            value: '&nbsp;<em>You clicked menu item 2!</em>'
+          }
+        ];
+        callback(items);
+      }
+    });
+  },
+  content_style: 'body { font-family:Helvetica,Arial,sans-serif; font-size:16px }'
+});
+----
+
+.Migrating to {productname} {productmajorversion}
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',
+  toolbar: 'myButton',
+  menubar: false,
+  setup: (editor) => {
+    editor.ui.registry.addSplitButton('myButton', {
+      text: 'My Button',
+      icon: 'info',
+      tooltip: 'Execute my action',
+      chevronTooltip: 'My Button menu options',
+      onAction: () => editor.insertContent('<p>You clicked the main button</p>'),
+      onItemAction: (api, value) => editor.insertContent(value),
+      fetch: (callback) => {
+        const items = [
+          {
+            type: 'choiceitem',
+            icon: 'notice',
+            text: 'Menu item 1',
+            value: '&nbsp;<em>You clicked menu item 1!</em>'
+          },
+          {
+            type: 'choiceitem',
+            icon: 'warning',
+            text: 'Menu item 2',
+            value: '&nbsp;<em>You clicked menu item 2!</em>'
+          }
+        ];
+        callback(items);
+      }
+    });
+  },
+  content_style: 'body { font-family:Helvetica,Arial,sans-serif; font-size:16px }'
+});
+----
+
+
+== Core API Changes
+
+[discrete]
+=== Breaking Changes Overview
+
+IMPORTANT: The following sections detail important changes that require updates to your code. Please review each section carefully.
+
+=== Updated Methods
+
+[[skipfocus-consolidation]]
+==== skipFocus and skip_focus
+// #TINY-12044
+
+The `skipFocus` and `skip_focus` options for the `ToggleToolbarDrawer` command have been consolidated into a single, more consistent argument. This reduces API complexity and clarifies the intended behavior.
+
+**Impact**: This change simplifies focus management, reducing the risk of confusion and unexpected behavior.
+
+**Migration steps:**
+
+[source, javascript]
+----
+// Old approach (Deprecated) in TinyMCE 8
+editor.execCommand('ToggleToolbarDrawer', { skip_focus: false }, { skipFocus: true });
```

Comment:
```suggestion
editor.execCommand('ToggleToolbarDrawer', false, { skipFocus: true })
```
The command doesn't look correct 🤔  
https://ephocks.atlassian.net/browse/TINY-12044
https://fiddle.tiny.cloud/ClRQ2cMIQk/1